app/views/adventures/authenticate_player_age.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="AccountForm Container">
+  <h1 class="PageBanner">View an<br>Age Protected Story</h1>
+  <%= form_for :adventure, url: authenticate_adventure_path do |f| %>
+    <div class="Field">
+      <%= f.label :age_limit, "Enter your age to access this story" %>
+      <%= f.number_field :age_limit, value: nil, step: :any %>
```

Comment:
For whatever reason I had it in my head that we should be collecting the user's birthday here instead of their age, and then running the calculation on the backend to figure out how old they are. But that seems extra complicated for little reason, and just as easy to lie about either way. So, this is solid.